calico_containers/nose.cfg

```diff
@@ -1,4 +1,4 @@
 [nosetests]
 with-coverage=1
 cover-erase=1
-cover-package=pycalico
+cover-package=calico_ctl
```

Comment:
There should be a way of specifying both pycalico and calico_ctl as cover-packages, probably like `cover-package=pycalico,calico_ctl`